variant.go

```diff
@@ -41,38 +41,54 @@ func (api *API) NewVariant() *Variant {
 	return &Variant{api: api}
 }
 
+func (obj *Variant) Get() (*Variant, error) {
+	return obj, requestVariant("GET", obj)
```

Comment:
do you have to return `obj` ? it seem to assign the Variant obj from the response at line 92 🤔